api/src/controllers/put/administrative-sanction.js

```diff
@@ -306,6 +337,7 @@ exports.editLNG = async function(args, res, next, incomingObj) {
  * @returns edited nrced administrativeSanction record
  */
 exports.editNRCED = async function(args, res, next, incomingObj) {
+  console.log('2', incomingObj);
```

Comment:
Remove.

Reply:
Bleh, Fill fix, and the other console logs